cns/restserver/internalapi.go

```diff
@@ -325,6 +342,24 @@ func (service *HTTPRestService) ReconcileNCState(ncRequest *cns.CreateNetworkCon
 	return 0
 }
 
+func podInfoByIPToPodInfoExtByInterfaceID(podInfoByIP map[string]cns.PodInfo) map[string]podInfoExt {
```

Comment:
is it possible to rename this func to something else that indicates the func?
something like GeneratePodInterfaceToPodIPsInfoMap ?

Reply:
yeah, it's a mouthful 😅. it's basically a "merge concat" operation (merging by same interface, concatenating all IPs for the same interface), but maybe a constructor like "newInterfaceToPodIPsInfoMap"?